build-system/tasks/clean.js

```diff
@@ -16,23 +16,27 @@
 'use strict';
 
 const del = require('del');
+const gulpCache = require('gulp-cache');
 
 /**
  * Clean up the build artifacts
  * @return {!Promise}
  */
 async function clean() {
-  return del([
-    'dist',
-    'dist.3p',
-    'dist.tools',
-    'build',
-    '.amp-build',
-    'deps.txt',
-    'EXTENSIONS_CSS_MAP',
-    'build-system/runner/build',
-    'build-system/runner/dist',
-    'build-system/server/new-server/transforms/dist',
+  return Promise.all([
+    gulpCache.clearAll(),
```

Comment:
To which directory does `gulp-cache` write its files? And why not add that directory to `.gitignore`?

Reply:
It stores it in the OS's tmp directory, so we don't need to worry about it.